src/lib/isTaxID.js

```diff
@@ -14,6 +22,386 @@ import assertString from './util/assertString';
  * for more information.
  */
 
+// Helper functions
+
+/*
+ * ISO 7064 validation helper function
+ * Called with an array of single-digit integers by locale-specific functions
+ * to validate TINs according to ISO 7064 (MOD 11, 10).
+ */
+function iso7064Check(digits) {
```

Comment:
We can move these general-purpose algo checks (up to L118) into the `src/lib/util/algorithms.js` and export and re-use here and any other place in future.

Reply:
agreed, will push both changes this weekend

Reply:
Let me know once you are done with the changes.